webrecorder/browsertrix
# Update org settings & org invite UI

Adds separate section for org members and updates invite form to be in a dialog. Resolves https://github.com/webrecorder/browsertrix-cloud/issues/527.

Note: Pending invites and updating user roles are not a part of this PR as they need some API changes ([see comment](https://github.com/webrecorder/browsertrix-cloud/issues/432#issuecomment-1406034527).) Visual style changes haven't been implemented at this time either, just some tweaks to get closer to the mockups.

### Manual testing
1. Log in and go to Org Settings page.
2. Go to "Org Information". Verify you're able to save a name as expected
3. Go to "Members" and click "Invite New Member". Verify you can invite a member as expected.

### Screenshots
<img width="1144" alt="Screen Shot 2023-01-26 at 9 25 59 PM" src="https://user-images.githubusercontent.com/4672952/215016120-c02ec181-217f-43f6-9153-eb9d4c84748b.png">
<img width="1135" alt="Screen Shot 2023-01-26 at 9 28 34 PM" src="https://user-images.githubusercontent.com/4672952/215016154-a44c2130-6dc6-4f9d-8ac2-a46900c0b8cc.png">
